regression/scripts/Vehicles/Tiltwing.py

```diff
@@ -260,7 +260,11 @@ def vehicle_setup():
     rot.design_altitude          = 500 * Units.feet                  
     Lift                         = vehicle.mass_properties.takeoff*9.81  
     rot.design_thrust            = (Lift * 1.5  )/net.number_of_engines  
-    rot.induced_hover_velocity   = np.sqrt(Lift/(2*rho*rot.disc_area*net.number_of_engines))                     
+    rot.induced_hover_velocity   = np.sqrt(Lift/(2*rho*rot.disc_area*net.number_of_engines))    
```

Comment:
Not for this PR. But this equation is not very SUAVE style.

Reply:
I agree.